### DISCUSSION TITLE
How to upload dataset submissions containing a large number of files and/or very big files

### DISCUSSION THREAD

**Discussion by ypriverol** [PRIDE-Archive/pride-resources#35] [Q&A]
This type of dataset submissions may fail while performing the submission using the PX Submission Tool. In those cases, the protocol to follow would be:

**Step 1** - Create "dummy" files with the exact names of your files in separate folder.

**Step 2** – Login to PX Submission Tool-> provide all metadata-> select the "dummy" files in the file selection and follow the steps.

---Please say no to checksum calculation step and go further. 

**Step 3** – At the end before finalising the submission, export the submission.px file and close the PX Submission Tool. You don’t have to go further.

**Step 4** – Now upload the submission.px file together with your original files to a dedicated manual upload area provided by PRIDE support.

**Note** : Please ask [pride-support@ebi.ac.uk](mailto:pride-support@ebi.ac.uk) for dedicated manual upload area if you have a large number of files or big files.

**Comment by deeptijk** [comment]
Step 4 – Now upload the submission.px file together with your "original files" to a dedicated manual upload area provided by PRIDE support.

**Reply by javizca** [reply]
yes, now corrected. Sorry
